openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -135,27 +136,121 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
   }
 
   .student__answer__display__content {
-    border: 1px solid rgba($color-decorative-staff, 0.25);
-    @include padding(($baseline-v/2), ($baseline-h/2), ($baseline-v/2), ($baseline-h/2));
+    @include padding(($baseline-v/2), 0, ($baseline-v/2), 0);
     margin-bottom: ($baseline-v/2);
   }
 
-  .openassessment__student-info_list {
+  .staff-info__student__report {
     list-style-type: none;
+
+    h2.title {
```

Comment:
remove h2 selector. add some padding and boldness:
```
padding: ($baseline-v/2) ($baseline-h/2);
font-weight: %t-strong;
```
and this needs the span fix as well: 

```
span {
font-weight: inherit;
}
```